### PR TITLE
Gui 2365 Magic search for metrics landing

### PR DESCRIPTION
**Pull request by dkavanagh**
Must merge PR#1758 before this one! These changes will be much easier to review after that.

https://eucalyptus.atlassian.net/browse/GUI-2365

### CODE REVIEW

**Comment by kamalgill** [comment]
Merged https://github.com/eucalyptus/eucaconsole/pull/1758

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1393/
Test FAILed.

File: eucaconsole/views/metrics.py [hunk 1]
```diff
@@ -28,6 +28,8 @@
 Pyramid views for Eucalyptus and AWS CloudWatch metrics
 
 """
+import copy
+import re
```
Comment (kamalgill) [inline eucaconsole/views/metrics.py]: @dkavanagh : This is an unused import

File: eucaconsole/views/metrics.py [hunk 2]
```diff
@@ -227,7 +316,8 @@ def cloudwatch_metrics_json(self):
                         res_ids=res_ids,
                         res_types=res_types,
                         unit=unit[0] if unit else '',
-                        metric_name=metric_name
+                        metric_name=metric_name,
+                        metric_label=METRIC_LABEL[metric_name] if metric_name in METRIC_LABEL else metric_name
```
Comment (kamalgill) [inline eucaconsole/views/metrics.py]: @dkavanagh : This line could be simplified as...
```
METRIC_LABEL.get(metric_name) or metric_name
```

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1397/
Test FAILed.

**Comment by aliceh** [comment]
Retest this please

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1398/
Test FAILed.

**Comment by aliceh** [comment]
Tests failed due to instabilities in Jenkins

**Pull request merged**